apps/portals/elportal/src/pages/HomePageV2.tsx

```diff
@@ -162,105 +162,129 @@ export default function HomePage() {
         }}
       >
         <ELGettingStarted />
+      </SectionLayout>
+      <SectionLayout
+        ContainerProps={{
+          className: 'home-spacer',
+          maxWidth: false,
+          style: { padding: 0 },
+        }}
+      >
```

Comment:
Each of these are their own "section" so they should each have their own "SectionLayout"

Reply:
![image](https://github.com/user-attachments/assets/8e467a2e-4edc-4596-a21d-583a1bdc8352)